### DISCUSSION TITLE
Able to build the vscode repo - but local changes are not reflected

### DISCUSSION THREAD

**Discussion by purplecomet1** [microsoft/vscode-discussions#94] [Extension Development QnA] [answered]
Hi,

I am able to build vscode from source to Desktop and Web, but any changes I make locally are not reflected when I rebuild. Any ideas why?

Thanks

**Comment by isidorn** [comment, marked as answer]
Make sure to follow these steps https://github.com/microsoft/vscode/wiki/How-to-Contribute
If something still does not work I suggest to open an issue here https://github.com/microsoft/vscode/issues
